xml/System.Windows.Controls/GridViewColumn.xml

```diff
@@ -364,11 +364,11 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Windows.Controls.GridViewColumn.DisplayMemberBinding%2A> property associates a type of data from a data source to a column so that the column displays a set of values of that type. Examples of data source objects include <xref:System.Windows.Data.XmlDataProvider> objects and <xref:System.Windows.Data.ObjectDataProvider> objects.  
+ This property associates a type of data from a data source to a column so that the column displays a set of values of that type. Examples of data source objects include <xref:System.Windows.Data.XmlDataProvider> objects and <xref:System.Windows.Data.ObjectDataProvider> objects.  
   
  The following properties are all used to define the content and style of a column cell, and are listed here in their order of precedence, from highest to lowest:  
   
--   <xref:System.Windows.Controls.GridViewColumn.DisplayMemberBinding%2A>  
+-   DisplayMemberBinding (this property) 
```

Comment:
nit: you might wanna add backticks since this is an API name (so locking for loc)